<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1580</width>
    <height>810</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1580</width>
    <height>810</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>RtBloodSmear</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>9</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>9</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>9</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>6</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="pushButton_process">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>140</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{border-radius: 4px;background-color: rgb(50, 157, 253);color: white;}QPushButton:hover{background-color: rgb(29, 148, 255);}QPushButton:pressed{background-color: rgb(10, 139, 255);}QPushButton:disabled{background-color: lightgray;}</string>
            </property>
            <property name="text">
             <string>process constol</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_debug">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>140</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{border-radius: 4px;background-color: rgb(50, 157, 253);color: white;}QPushButton:hover{background-color: rgb(29, 148, 255);}QPushButton:pressed{background-color: rgb(10, 139, 255);}QPushButton:disabled{background-color: lightgray;}</string>
            </property>
            <property name="text">
             <string>debug info</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_sensor">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>140</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{border-radius: 4px;background-color: rgb(50, 157, 253);color: white;}QPushButton:hover{background-color: rgb(29, 148, 255);}QPushButton:pressed{background-color: rgb(10, 139, 255);}QPushButton:disabled{background-color: lightgray;}</string>
            </property>
            <property name="text">
             <string>sensor monitor</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_stainpool">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>140</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{border-radius: 4px;background-color: rgb(50, 157, 253);color: white;}QPushButton:hover{background-color: rgb(29, 148, 255);}QPushButton:pressed{background-color: rgb(10, 139, 255);}QPushButton:disabled{background-color: lightgray;}</string>
            </property>
            <property name="text">
             <string>stain pool</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>951</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>2</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>2</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(197, 197, 197);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <widget class="DeviceSettingPage" name="page_device"/>
      <widget class="SensorCheckPage" name="page_sensor"/>
      <widget class="DebugRecordPage" name="page_debug"/>
      <widget class="StainPoolPage" name="page_stain"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1580</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_function">
    <property name="title">
     <string>function</string>
    </property>
    <addaction name="action_debug"/>
    <addaction name="action_process"/>
    <addaction name="action_information"/>
   </widget>
   <addaction name="menu_function"/>
  </widget>
  <action name="action_debug">
   <property name="text">
    <string>debug record</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="action_process">
   <property name="text">
    <string>process</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="action_information">
   <property name="text">
    <string>information</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DeviceSettingPage</class>
   <extends>QWidget</extends>
   <header>views/device_setting_page.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DebugRecordPage</class>
   <extends>QWidget</extends>
   <header>views/debug_record_page.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SensorCheckPage</class>
   <extends>QWidget</extends>
   <header>views/sensor_check_page.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>StainPoolPage</class>
   <extends>QWidget</extends>
   <header>views/stain_pool_page.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
